src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jitsi.impl.neomedia;
+package org.jitsi.service.neomedia;
 
 
-import org.jitsi.service.neomedia.*;
+import org.jitsi.impl.neomedia.*;
```

Comment:
Stuff from .service. shouldn't reference anything inside .impl. to truly follow the service vs. impl packaging